config/crossSections.yaml

```diff
@@ -0,0 +1,748 @@
+  unc: 4263.0
+QCD_Pt-50To80_MuEnrichedPt5:
+  crossSec: 377800.0
+  reference: XSDB QCD_Pt-50To80_MuEnrichedPt5_TuneCP5_13TeV-pythia8 modifiedOn 2021-06-21 09:52:30
+  unc: 1184.0
+QCD_Pt-80To120_MuEnrichedPt5:
+  crossSec: 88620.0
+  reference: XSDB QCD_Pt-80To120_MuEnrichedPt5_TuneCP5_13TeV-pythia8 modifiedOn 2021-06-21 09:52:36
+  unc: 275.1
+QCD_Pt-120To170_MuEnrichedPt5:
+  crossSec: 21070.0
+  reference: XSDB QCD_Pt-120To170_MuEnrichedPt5_TuneCP5_13TeV-pythia8 modifiedOn 2021-06-21 09:52:17
+  unc: 65.28
+QCD_Pt-170To300_MuEnrichedPt5:
+  crossSec: 7019.0
+  reference: XSDB QCD_Pt-170To300_MuEnrichedPt5_TuneCP5_13TeV-pythia8 modifiedOn 2021-06-21 09:52:21
+  unc: 21.61
+QCD_Pt-300To470_MuEnrichedPt5:
+  crossSec: 622.4
+  reference: XSDB QCD_Pt-300To470_MuEnrichedPt5_TuneCP5_13TeV-pythia8 modifiedOn 2021-06-21 09:52:21
+  unc: 1.891
+QCD_Pt-470To600_MuEnrichedPt5:
+  crossSec: 58.86
+  reference: XSDB QCD_Pt-470To600_MuEnrichedPt5_TuneCP5_13TeV-pythia8 modifiedOn 2021-06-21 09:52:28
+  unc: 0.1776
+QCD_Pt-600To800_MuEnrichedPt5:
+  crossSec: 18.22
+  reference: XSDB QCD_Pt-600To800_MuEnrichedPt5_TuneCP5_13TeV-pythia8 modifiedOn 2021-06-21 09:52:32
+  unc: 0.05471
+QCD_Pt-800To1000_MuEnrichedPt5:
+  crossSec: 3.25
+  reference: XSDB QCD_Pt-800To1000_MuEnrichedPt5_TuneCP5_13TeV-pythia8 modifiedOn 2021-06-21 09:52:34
+  unc: 0.0148
+QCD_Pt-1000ToInf_MuEnrichedPt5:
+  crossSec: 1.078
+  reference: XSDB QCD_Pt-1000toInf_MuEnrichedPt5_TuneCP5_13TeV_pythia8 modifiedOn 2020-09-24 04:59:49
+  unc: 0.003236
+QCD_Pt-15to20_EMEnriched:
+  crossSec: 1324000.0
+  reference: XSDB QCD_Pt-15to20_EMEnriched_TuneCP5_13TeV-pythia8 modifiedOn 2022-03-02 06:42:14
+  unc: 4183.0
+QCD_Pt-20to30_EMEnriched:
+  crossSec: 4896000.0
+  reference: XSDB QCD_Pt-20to30_EMEnriched_TuneCP5_13TeV-pythia8 modifiedOn 2022-03-02 06:42:16
+  unc: 15410.0
+QCD_Pt-30to50_EMEnriched:
+  crossSec: 6447000.0
+  reference: XSDB QCD_Pt-30to50_EMEnriched_TuneCP5_13TeV-pythia8 modifiedOn 2021-10-11 11:25:28
+  unc: 19870.0
+QCD_Pt-50to80_EMEnriched:
+  crossSec: 1988000.0
+  reference: XSDB QCD_Pt-50to80_EMEnriched_TuneCP5_13TeV-pythia8 modifiedOn 2021-10-11 11:25:29
+  unc: 5961.0
+QCD_Pt-80to120_EMEnriched:
+  crossSec: 367500.0
+  reference: XSDB QCD_Pt-80to120_EMEnriched_TuneCP5_13TeV-pythia8 modifiedOn 2021-10-11 11:25:30
+  unc: 1091.0
+QCD_Pt-120to170_EMEnriched:
+  crossSec: 66590.0
+  reference: XSDB QCD_Pt-120to170_EMEnriched_TuneCP5_13TeV-pythia8 modifiedOn 2021-10-11 11:25:25
+  unc: 196.6
+QCD_Pt-170to300_EMEnriched:
+  crossSec: 16620.0
+  reference: XSDB QCD_Pt-170to300_EMEnriched_TuneCP5_13TeV-pythia8 modifiedOn 2021-10-11 11:25:26
+  unc: 49.1
+QCD_Pt-300toInf_EMEnriched:
+  crossSec: 1104.0
+  reference: XSDB QCD_Pt-300toInf_EMEnriched_TuneCP5_13TeV-pythia8 modifiedOn 2021-10-11 11:25:28
+  unc: 3.282
+DY_NNLO:
+    crossSec: 6077.22
+    reference: XSDB DYJetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2019-10-11 14:36:12
+    unc: 2% (to be corrected)
```

Comment:
maybe a better way to treat correctly uncertianties % when we load it (to keep config simple) => keep it as "2%"

Reply:
ok